web/src/features/moreCast2/interfaces.tsx

```diff
@@ -0,0 +1,19 @@
+import { DateTime } from 'luxon'
```

Comment:
Minor nit: since there's no react components here we could rename this from `.tsx` to `.ts`